docs/java-troubleshooting.md

```diff
@@ -27,3 +27,9 @@ You can find further information in [Scope Java agent - Running tests inside a c
 If you are using certain libraries or version libraries that are not officially supported by the Scope Java agent, you might have not see data in Scope after executing the test phase.
 
 You can find further information in [Scope Java agent - Compatibility](java-compatibility.md) section.
+
+## I'm getting `SSLException: Received close_notify during handshake` error:
+
+This error may occur if you are using an outdated JDK 1.7 patch version due to some required cipher suites are not supported in that version.
```

Comment:
the phrase is broken.
Either
```suggestion
This error may occur if you are using an outdated JDK 1.7 patch version because some required cipher suites are not supported in that version.
```
or
```suggestion
This error may occur if you are using an outdated JDK 1.7 patch version due to some required cipher suites that are not supported in that version.
```

I'd go for the first one